content/_index/overview/memoryinspector.md

```diff
@@ -0,0 +1,10 @@
++++
+title = "Memory Inspector"
+weight = 20
+
+[asset]
+  icon = "fas magnifying-glass"
```

Comment:
```suggestion
  icon = "fas fa-search"
```

It looks like this Hugo theme doesn't support all fontawesome icons, see https://github.com/eclipsesource/syna/blob/master/assets/styles/fontawesome/_icons.scss. How about the search icon instead?